### DISCUSSION TITLE
Option for Graph/map view?

### DISCUSSION THREAD

**Discussion by garrettvercoe** [crazko/krystal#37] [Ideas]
Hi! Wondering if anyone has gotten a map/connected graph view working with this plugin.

I've tried TiddlyMap but it ended up breaking the behavior. If anyone has any workarounds or alternatives, it would be much appreciated 😄

**Comment by crazko** [comment]
Hello there. So far, I do not have any satisfying solution for that, but you can [have a look at my comment](https://github.com/crazko/krystal/issues/6#issuecomment-649807104), where I describe one way to produce a graph of tiddlers in one visualization tool.

**Reply by arunnbabu81** [reply]
Currently i am using [this method](https://talk.tiddlywiki.org/t/how-to-create-a-viewtemplate-to-float-the-thebrain-echarts-to-the-right-of-tiddler-text-field-using-shiraz-flexgrid-autolayout-with-image/2497) to use the graph view in krystal along with the tiddler body text side by side.
